<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfSensorVolumesPrefWidget</class>
 <widget class="QWidget" name="WkfSensorVolumesPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>传感器体积</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>体积类型</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="mVolumeType">
     <property name="currentText">
      <string>转动限制</string>
     </property>
     <item>
      <property name="text">
       <string extracomment="Displays the mechanical limits of the components articulation.">Slew Limits</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>引导限制</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>扫描限制</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>视野</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>波束宽度</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>计算</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>传感器模式</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="sensorTableWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>模式</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>面</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>边</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>投影</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addSensorButton">
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteSensorButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>武器模式</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="weaponTableWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>模式</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>面</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>边</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>投影</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="addWeaponButton">
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteWeaponButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
